modules/services/localization/src/main/res/values/strings.xml

```diff
@@ -399,6 +399,7 @@
     <string name="player_play_failed_check_internet">Could not load podcast. Please check your internet connection and try again.</string>
     <string name="player_played_up_to">Played up to %s</string>
     <string name="player_time_remaining">Time remaining %s</string>
+    <string name="swipe_affordance_content_description">Swipe Affordance</string>
```

Comment:
```suggestion
    <string name="swipe_affordance_content_description">Drag down to dismiss</string>
```
Feel free to ignore this, but I wonder if it would be better to state what action can be taken.

Reply:
I like that [5cf6b4b](https://github.com/Automattic/pocket-casts-android/pull/3151/commits/5cf6b4b0273e537e7d6673c0fa373db55b21add5)